app/core.go

```diff
@@ -1775,6 +1775,9 @@ func (a *App) formatChangeInfoLink(ch *gerrit.ChangeInfo) string {
 
 func (a *App) prepareUserLink(ctx context.Context, account *events.Account) string {
 	chatID := a.lookupGerritUser(ctx, account)
+	if a.isBlocklisted(ctx, chatID) {
+		return chatID
```

Comment:
`chatID` will be a string like `"U037Q3NGS9B"`, which will be unhelpful to the average reader. We should return `account.DisplayName()` instead.